frontend/src/pages/BookPage.js

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+import PReadsNavbar from '../components/PReadsNavbar';
+import Homefeed from '../components/Homefeed';
+import Exercises from '../components/Exercises';
+import Readings from '../components/Readings';
+import Games from '../components/Games';
+import Results from '../components/Results';
+import Catalog from '../components/Catalog';
+import Login from '../components/Login';
+import CatalogCard from '../components/CatalogCard';
+import BookInfo from '../components/BookInfo';
+import QuizViewer from '../components/QuizViewer';
+import { getBookData, getQuizzes } from '../utils/api';
+import { Button } from 'reactstrap';
+import '../styles/BookPage.scss';
+class BookPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bookID: props.match.params.id,
+      bookData: getBookData(props.match.params.id),
+      quizData: getQuizzes(props.match.params.id),
+      quizIndex: -1,
+      quizID: -1,
+      currentQuestions: []
+    };
+  }
+
+  navigationOptions = [
+    {
+      route: '/kids',
+      name: 'Home',
+      component: Homefeed
+    },
+    {
+      route: '/kids/exercises',
+      name: 'Exercises',
+      component: Exercises
+    },
+    {
+      route: '/kids/readings',
+      name: 'Readings',
+      component: Readings
+    },
+    {
+      route: '/kids/games',
+      name: 'Games',
+      component: Games
+    },
+    {
+      route: '/kids/results',
+      name: 'Results',
+      component: Results
+    },
+    {
+      // TODO: remove in future
+      route: '/kids/catalog',
+      name: 'Catalog',
+      component: Catalog
+    }
+  ];
+
+  getCards = () => {
+    var cards = [];
+    for (var i in this.state.quizData) {
+      cards.push({
+        title: this.state.quizData[i]['name'],
+        id: this.state.quizData[i]['id'],
+        index: i
+      });
+    }
+    return cards;
+  };
+
+  selectQuiz = (id, index) => {
```

Comment:
not sure if easily possible but scrolling down to the newly opened quiz could be cool so the user isn't confused if they click the button and "nothing" happens